nodejs/sample-apps/aws-sdk/deploy/main.tf

```diff
@@ -17,10 +17,11 @@ module "hello-lambda-function" {
   ])
 
   environment_variables = {
-    AWS_LAMBDA_EXEC_WRAPPER = "/opt/otel-handler"
-    OTEL_TRACES_EXPORTER    = "logging"
-    OTEL_METRICS_EXPORTER   = "logging"
-    OTEL_LOG_LEVEL          = "DEBUG"
+    AWS_LAMBDA_EXEC_WRAPPER     = "/opt/otel-handler"
+    OTEL_TRACES_EXPORTER        = "logging"
+    OTEL_METRICS_EXPORTER       = "logging"
+    OTEL_LOG_LEVEL              = "DEBUG"
+    OTEL_EXPORTER_OTEL_ENDPOINT = "http://localhost:55681/v1/traces"
```

Comment:
This variable has a typo which was fixed: https://github.com/open-telemetry/opentelemetry-lambda/commit/3f5d7b8bcbc5d2b9cb6283fdb7c772414045e90b